public/app/features/explore/ExploreQueryInspector.tsx

```diff
@@ -57,7 +57,7 @@ export function ExploreQueryInspector(props: Props) {
       <InspectDataTab
         data={dataFrames}
         dataName={'Explore'}
-        isLoading={loading}
+        isLoading={queryResponse.state === LoadingState.Loading}
```

Comment:
Nice 🚀  tested Explore and it's now possible to use the inspector in Live mode 🙌